app/commands/codemod/confirm.go

```diff
@@ -0,0 +1,157 @@
+package codemod
+
+import (
+	"bufio"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/codelingo/lingo/app/util"
+	flow "github.com/codelingo/platform/flow/rpc/flowengine"
+	"github.com/fatih/color"
+	"github.com/waigani/diffparser"
+)
+
+type IssueConfirmer struct {
+	keepAll bool
+	output  bool
+}
+
```

Comment:
Boolean arguments can indicate low cohesion. Consider using a separate function for each case and helper functions for repeated code. This will make each function clearer and more modular, leading to easier maintainability.